R/stat-density.R

```diff
@@ -23,6 +23,8 @@
 #' @eval rd_computed_vars(
 #'  density  = "density estimate.",
 #'  count    = "density * number of points - useful for stacked density plots.",
+#'  wdensity = "density * sum of weights. In absence of weights, the same as
+#'  `count`",
```

Comment:
Probably better to add a period at the end of this sentence as is the case for all the other descriptions.

Reply:
Ah yes, thanks for catching!